### PR TITLE
Log instances of excessive I/O waits

### PR DESCRIPTION
**Pull request by achamayou**
Tried to get cimetrics working with this, but it needs more YAML to work than I want to spend time writing (because of the Daily), so I've deleted that, and it's just logging now.

### CODE REVIEW

File: .azure-pipelines-templates/test.yml [hunk 1]
```diff
@@ -66,3 +66,18 @@ steps:
       testRunTitle: "${{ parameters.suite_name_suffix }}"
       platform: "$(Agent.MachineName)"
     condition: succeededOrFailed()
+
+  - script: |
+      echo "=========== TOOK TOO LONG "
```
Comment (achamayou) [inline .azure-pipelines-templates/test.yml:71]:
```suggestion
      apt install ripgrep
      echo "TOOK TOO LONG "
```

File: .azure-pipelines-templates/test.yml [hunk 2]
```diff
@@ -66,3 +66,19 @@ steps:
       testRunTitle: "${{ parameters.suite_name_suffix }}"
       platform: "$(Agent.MachineName)"
     condition: succeededOrFailed()
+
+  - script: |
+      apt install ripgrep
```
Comment (achamayou) [inline .azure-pipelines-templates/test.yml:71]:
```suggestion
      sudo apt install ripgrep
```

File: .azure-pipelines-templates/misc.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+parameters:
+  suite_name_suffix: ""
+
+steps:
+  # Whether tests pass or fail, print out a per-logfile, and total number of I/O operations that
+  # exceeded our opinion of "too long".
+  - script: |
+      sudo apt install ripgrep > /dev/null
```
[the remark this answers is not in the document]
Comment (achamayou) [reply .azure-pipelines-templates/misc.yml:8]: I thought of doing that, but it's consistently <1s, so I don't see the point.

**Review by wintersteiger — APPROVED** [review]
LGTM!

**Pull request merged**